doc/proxy-clients.rst

```diff
@@ -97,12 +97,28 @@ You can also pass some options to the Varnish client:
 * ``tags_header`` (X-Cache-Tags): Allows you to change the HTTP header used for
   tagging. If you change this, make sure to use the correct header name in your
   :doc:`proxy server configuration <proxy-configuration>`;
+* ``tags_purge_header`` (key): Allows you to change the HTTP header used for
```

Comment:
what is this header? is it in addition to tags_header? if its instead of the other tags_header, then why not use the tags_header configuration for it?